javascripting/array-filtering.js

```diff
@@ -0,0 +1,7 @@
+var numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+var filtered = numbers.filter(function(number) {
```

Comment:
While your method is perfectly fine. You can also do: `numbers.filter(x => x % 2 == 0);` using the ES6 arrow functions.